packages/website/public/index.html

```diff
@@ -1,53 +1,44 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <meta name="description" content="An Open Protocol For Decentralized Exchange On The Ethereum Blockchain" />
-        <meta property="og:type" content="website" />
-        <meta property="og:title" content="0x" />
-        <meta property="og:description" content="An Open Protocol For Decentralized Exchange On The Ethereum Blockchain" />
-        <meta property="og:image" content="/images/og_image.png" />
-        <title>0x: The Protocol for Trading Tokens</title>
-        <link rel="icon" type="image/png" href="/images/favicon/favicon-2-32x32.png" sizes="32x32" />
-        <link rel="icon" type="image/png" href="/images/favicon/favicon-2-16x16.png" sizes="16x16" />
-        <link rel="stylesheet" href="/css/atom-one-light.css">
-        <link rel="stylesheet" href="/css/material-design-iconic-font.min.css">
-        <link rel="stylesheet" href="/css/roboto.css">
-        <link rel="stylesheet" href="/css/roboto_mono.css">
-        <link rel="stylesheet" href="/css/basscss_responsive_custom.css">
-        <link rel="stylesheet" href="/css/basscss_responsive_padding.css">
-        <link rel="stylesheet" href="/css/basscss_responsive_margin.css">
-        <link rel="stylesheet" href="/css/basscss_responsive_type_scale.css">
-    </head>
-    <body style="margin: 0px; min-width: 355px;">
-        <!-- Google Analytics -->
-        <script>
-            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-            })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
 
-            ga('create', 'UA-98720122-1', 'auto');
-            ga('send', 'pageview');
-        </script>
-        <!-- End Google Analytics -->
-        <!-- Facebook SDK -->
-        <div id="fb-root"></div>
-        <script>
-            (function(d, s, id) {
-                var js, fjs = d.getElementsByTagName(s)[0];
-                if (d.getElementById(id)) return;
-                js = d.createElement(s); js.id = id;
-                js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.8&appId=1687545238205192";
-                fjs.parentNode.insertBefore(js, fjs);
-            }(document, 'script', 'facebook-jssdk'));
-        </script>
-        <div id="app"></div>
-        <!-- End Facebook SDK -->
-        <!-- Twitter SDK -->
-        <script>
-            window.twttr = (function(d, s, id) {
+<head>
```

Comment:
Why did we left-shift that section?

Reply:
Ran prettier again, and this is the same output